src/app/app.component.spec.ts

```diff
@@ -19,17 +19,4 @@ describe('AppComponent', () => {
     const app = fixture.componentInstance;
     expect(app).toBeTruthy();
   });
```

Comment:
Can we add a simple test for the menu?
I think it is really important that from the beginning we create as much as possible tests.

Reply:
yes, you are absolutely right,  I will add some tests here